### PR TITLE
Fix Typos

### PR DESCRIPTION
**Pull request by saurabhuic**

### CODE REVIEW

File: examples/use-cases/Instruments.ipynb [hunk 1]
```diff
@@ -1430,7 +1430,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "In addition to upserting instruments we can also delete them. When we delete an instrument it becomes unvailable for us to use from the time that we delete it onwards. If we want to look back in time at our instruments or transactions in those instruments we will be able to find them, they are not permanently deleted from the system.\n",
```
Comment (victorfinbourne) [inline examples/use-cases/Instruments.ipynb:1433]: unavailable

File: examples/use-cases/Transparency - Multiple Systems.ipynb [hunk 2]
```diff
@@ -550,7 +550,7 @@
     "\n",
     "As part of this definition you will attach both unique and non-unique identifiers to your instruments. Read more about identifiers here [LUSID Knowledge Base: Which Instrument Identifier Scheme Should I Use?](https://support.lusid.com/which-instrument-identifier-schemes-should-i-use-with-lusid).\n",
     "\n",
-    "You use an upsert method to add instrument definitions to the instrument universe in LUSID. Read more about the behaviour of the upsert method here [LUSID Knowledge Base: Upsert](https://support.lusid.com/upsert-command).\n",
+    "You use an upsert method to add instrument definitions to the instrument universe in LUSID. Read more about the behavior of the upsert method here [LUSID Knowledge Base: Upsert](https://support.lusid.com/upsert-command).\n",
```
Comment (victorfinbourne) [inline examples/use-cases/Transparency - Multiple Systems.ipynb:553]: is the documentation supposed to be in american english or british english?

File: examples/use-cases/Instruments.ipynb (hunk 1 above)
Comment (slemasne) [reply examples/use-cases/Instruments.ipynb:1433]: Now lives here:

https://github.com/finbourne/sample-notebooks/blob/master/examples/use-cases/ibor/Maintaining%20an%20instrument%20master%20in%20LUSID.ipynb

File: examples/use-cases/Transparency - Multiple Systems.ipynb (hunk 2 above)
Comment (slemasne) [reply examples/use-cases/Transparency - Multiple Systems.ipynb:553]: Pretty sure this should be British english.
@rizwansaeed ?

Comment (slemasne) [reply examples/use-cases/Transparency - Multiple Systems.ipynb:553]: Confirmed we should use British English.

**Comment by cscialino** [comment]
Closing PR and reopening after rebasing with current version of `develop`

**Pull request closed without merging**